### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Recursively**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -27,9 +27,9 @@
     t.string   "day"
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
-    t.integer  "create_mail_id"
     t.integer  "kps_route_id"
     t.boolean  "discontinued"
+    t.integer  "create_mail_id"
```
Comment (TanaTanoi) [inline db/schema.rb]: kinda confused about dis, if references are put at the bottom, should the kps_route go there too? or is it arbitrary?

Comment (Recursively) [inline db/schema.rb]: when I ran db:migrate this file got recreated. Not sure about this tbh

File: app/views/events/_update_cost.html.erb [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-  <%= "Frequency: #{@event.frequency} hours" %>
-  <%= "Weekly ship start: #{@event.day} at 6pm" %>
-</p>
\ No newline at end of file
+       by #{@event.company}" %></p>
+<p><%= "Company: #{@event.company}" %></p>
+<p><%= "Priority type: #{@event.priority}" %></p>
+<p><%= "Weight cost: #{@event.weight_cost}" %></p>
+<p><%= "Volume cost: #{@event.volume_cost}" %></p>
+<p><%= "Duration: #{@event.duration} hours" %></p>
```
Comment (divyap1) [inline app/views/events/_update_cost.html.erb]: I'm not sure how these are stored, but there is a price_format method in the application helper that will format prices to 2dp and place a $ in front, and maybe should be used here?

File: app/views/events/_update_price.html.erb [hunk 3]
```diff
@@ -1,7 +1,5 @@
...
-  <%= "Priority: #{@event.priority}" %>
-  <%= "Weight price: #{@event.weight_price}" %>
-  <%= "Volume price: #{@event.volume_price}" %>
-</p>
\ No newline at end of file
+<p><%= "Updated route price between #{@event.origin.city}, #{@event.origin.country}
+     and #{@event.destination.city}, #{@event.destination.country}" %></p>
+<p><%= "Priority: #{@event.priority}" %></p>
+<p><%= "Weight price: #{@event.weight_price}" %></p>
+<p><%= "Volume price: #{@event.volume_price}" %></p>
```
Comment (divyap1) [inline app/views/events/_update_price.html.erb]: Again the formatting for prices thing...

File: app/views/events/_update_cost.html.erb (hunk 2 above)
Comment (Recursively) [inline app/views/events/_update_cost.html.erb]: 👍

File: app/controllers/company_routes_controller.rb [hunk 4]
```diff
@@ -2,14 +2,14 @@ class CompanyRoutesController < ApplicationController
   before_action :set_company_route, only: [:destroy, :update, :edit, :show]
 
   def index
-    @routes = CompanyRoute.where(discontinued: false)
-    @kps_routes = KpsRoute.all
-    @discontinued_routes = CompanyRoute.where(discontinued: true)
+    @routes = CompanyRoute.where(discontinued: false).paginate(:page => params[:page], :per_page => 10)
```
Comment (TanaTanoi) [inline app/controllers/company_routes_controller.rb]: maybe make 10 into constants? `DEFAULT_PER_PAGE` or something?

File: app/controllers/events_controller.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 class EventsController < ApplicationController
   def index
-    @events = Event.all
+    @events = Event.paginate(:page => params[:page], :per_page => 9)
```
Comment (TanaTanoi) [inline app/controllers/events_controller.rb]: same for this one I guess, with a nice name (assuming 9 cos events use cards?)

**Pull request merged**